packages/vaex-core/vaex/dataframe.py

```diff
@@ -2686,6 +2686,44 @@ def f(i1, i2):
                 previous_chunk = previous.result()
                 yield previous_l1, previous_l2, previous_chunk
 
+    @docsubst
+    def to_records(self, index=None, selection=None, column_names=None, strings=True, virtual=True, parallel=True,
+                   chunk_size=None, array_type='python'):
+        """Return a list of [{column_name: value}, ...)] "records" where each dict is an evaluated row.
```

Comment:
to make the substitution not work here:
```suggestion
        """Return a list of [{{column_name: value}}, ...)] "records" where each dict is an evaluated row.
```